handlers/auth.ts

```diff
@@ -90,6 +107,18 @@ export const authCreate: Handler<T.Paths.AuthCreate.RequestBody> = async (c, _re
       handlePayload
     );
 
+    // Validate the expiration and the signature before submitting them
+    const blockNum = await getCurrentBlockNumber();
+    if (blockNum > handlePayload.expiration) return res.status(409).send();
+
+    const claimHandlePayload = api.registry.createType("ClaimHandle", handlePayload);
```

Comment:
@wesbiggs If you do this, you can remove all the registry code:

```suggestion
    const claimHandlePayload = api.registry.createType("CommonPrimitivesHandlesClaimHandlePayload", handlePayload);
```

All those types are there, but generated. So the names are not always what they appear.

Reply:
Ah, thanks. Saw that in the generated code but wasn't sure how it related.